src/unitxt/operators.py

```diff
@@ -658,6 +665,65 @@ def process(self, stream: Stream, stream_name: str = None) -> Generator:
                 yield instance
 
 
+class FilterByListsOfValues(SingleStreamOperator):
```

Comment:
Best to  it's better to keep it consistent.    I think the action we want to is filter the dataset.  I'll change the name of the parameter to 'required_values' in both classes, so it will be clear.